matrix_sdk_crypto/src/verification/sas/sas_state.rs

```diff
@@ -1394,7 +1394,7 @@ mod test {
 
         *method = StartMethod::Custom(CustomStartContent {
             method: "m.sas.custom".to_string(),
-            fields: vec![].into_iter().collect(),
+            data: vec![].into_iter().collect(),
```

Comment:
Could also write `BTreeMap::new()` (or `Default::default()` if you don't want the extra import).

Reply:
Thanks, updated.